draft-ietf-masque-connect-ethernet.md

```diff
@@ -465,14 +468,6 @@ and the Proxy is the same as in {{fig-full-tunnel}} above.
 
 # Performance Considerations
 
-When the protocol running inside the tunnel uses congestion control (e.g.,
```

Comment:
I'd recommend keeping the first and third sentences, and only drop the second one

Reply:
Done.